gulp/tasks/pages.js

```diff
@@ -15,6 +15,8 @@ module.exports = function (gulp, $, config) {
   var baseDir            = config.basePaths.src;
   var moduleHelpers      = pageshelpers(config);
 
+  const isProd = process.env.NODE_ENV === 'production';
```

Comment:
Let's move this to the config file instead of duplicating it all the time.
We use the config file to share the data across scripts